src/Object/P/Record.ts

```diff
@@ -0,0 +1,68 @@
+import {Modx} from '../_Internal'
+import {IterationOf} from '../../Iteration/IterationOf'
+import {Iteration} from '../../Iteration/Iteration'
+import {Pos} from '../../Iteration/Pos'
+import {Next} from '../../Iteration/Next'
+import {Key} from '../../Any/Key'
+import {LastIndex} from '../../List/LastIndex'
+import {List} from '../../List/List'
+
+/**
+ * @hidden
+ */
+type _RecordReqR<Path extends List<Key>, A, I extends Iteration = IterationOf<'0'>> =
```

Comment:
Should these internal types start with an underscore? In other places, like `O.Merge`, there is no underscore.

Reply:
No, it's not necessary. I use underscores to symbolize "steps" in the implementation (that sometimes it is excessively hard to name).